gpkit/tests/t_examples.py

```diff
@@ -82,6 +83,37 @@ def test_simpleflight(self, example):
             sol_rat = sol["sensitivities"]["constants"][key]/consenscheck[key]
             self.assertTrue(abs(1-sol_rat) < 1e-2)
 
+    # TODO: uncomment when breakdown is merged
+    # def test_breakdown_example(self, example):
+    #     if gpkit.units:
+    #         self.assertAlmostEqual(mag(example.sol["cost"]), 8.448, 3)
+    #         self.assertAlmostEqual(mag(example.sol("w2")), 6.448, 3)
+    #     else:
+    #         self.assertAlmostEqual(example.sol["cost"], 5, 4)
+    #         self.assertAlmostEqual(example.sol("w2"), 3, 4)
+
+    def test_LCS_ex(self, example):
+        self.assertAlmostEqual(example.sol["cost"], .5)
+
+    def test_subinplace(self, example):
+        self.assertAlmostEqual(example.sol["cost"], .5)
+
+    def test_dual_infeasible_ex(self, example):
+        with self.assertRaises(RuntimeWarning):
+            example.m.solve(verbosity=0)
```

Comment:
would be nice here to check for "DUAL_INFEAS_CER" if the solver is mosek; @bqpd do we have an easy way to check what the solver was/is for a `GeometricProgram` (in this case, we would need it for a failed solve, so attaching it to the gp makes more sense than putting it in the result)